.github/workflows/try-setup-install.yml

```diff
@@ -0,0 +1,37 @@
+name: Try Setup Install
+
+on: [push]
+
+jobs:
+
+  gather-py3-versions:
+    runs-on: ubuntu-latest
+    outputs:
+      matrix: ${{ steps.py3_versions.outputs.matrix }}
+    steps:
+      - uses: actions/checkout@v2
+      - name: Python Versions
+        id: py3_versions
+        run: |
+          minmin_maxmax=$(grep -P "\(\(\d+, ?\d+\), ?\(\d+, ?\d+\)\)" -oh setup.py | sed 's/[^0-9]/ /g')
+          IFS=', ' read -r -a array <<< "$minmin_maxmax"
+          min_thru_max_series=$(for i in `seq ${array[1]} ${array[3]}`; do printf "3.$i,"; done | rev | cut -c 2- | rev)
+          echo ::set-output name=matrix::{\"py3_versions\":[$(echo $min_thru_max_series)]}\"
+
+  setup-py-install:
+    needs: gather-py3-versions
+    runs-on: ubuntu-latest
+
+    strategy:
+      max-parallel: 4
+      fail-fast: false
+      matrix: ${{ fromJSON(needs.gather-py3-versions.outputs.matrix) }}
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.py3_versions }}
+      - run: |
+          echo ${{ matrix.py3_versions }}
+          python setup.py install
```

Comment:
```suggestion
name: Try Setup Install

on: [push]

jobs:

  gather-py3-versions:
    runs-on: ubuntu-latest
    outputs:
      matrix: ${{ steps.py3_versions.outputs.matrix }}
    steps:
      - uses: actions/checkout@v2
      - name: Python Versions
        id: py3_versions
        run: |
          minmin_maxmax=$(grep -P "\(\(\d+, ?\d+\), ?\(\d+, ?\d+\)\)" -oh setup.py | sed 's/[^0-9]/ /g')
          IFS=', ' read -r -a array <<< "$minmin_maxmax"
          min_thru_max_series=$(for i in `seq ${array[1]} ${array[3]}`; do printf "'3.$i',"; done | rev | cut -c 2- | rev)
          echo ::set-output name=matrix::{\"py3_versions\":[$(echo $min_thru_max_series)]}\"
          echo $min_thru_max_series

  pip-install:
    needs: gather-py3-versions
    runs-on: ubuntu-latest

    strategy:
      max-parallel: 4
      fail-fast: false
      matrix: ${{ fromJSON(needs.gather-py3-versions.outputs.matrix) }}

    steps:
      - uses: actions/checkout@v2
      - uses: actions/setup-python@v2
        with:
          python-version: ${{ matrix.py3_versions }}
      - run: |
          pip install --upgrade pip wheel setuptools
          pip install --editable .

```

Reply:
I'm not sure what's up with the diff here, but it's a copy of https://github.com/WIPACrepo/rest-tools/blob/master/.github/workflows/try-setup-install.yml